src/serializer/ResidualHeapSerializer.js

```diff
@@ -695,6 +696,99 @@ export class ResidualHeapSerializer {
     return t.arrayExpression(initProperties);
   }
 
+  _getPropertyValue(object: any, key: string, serialzeValue: boolean) {
```

Comment:
Avoid `any` whenever possible. Please plug in correct type.
It seems that a better name for `object` would be `properties`.
Spelling of `serialzeValue`.

Reply:
I opted for `_getSerializedPropertyValue(properties: Map<string, any>, key: string) {` instead, is that okay for this?